Rlabkey/R/labkey.experiment.R

```diff
@@ -171,3 +171,30 @@ labkey.experiment.saveBatch <- function(baseUrl=NULL, folderPath, assayConfig =
 
 	return (fromJSON(response, simplifyVector=FALSE, simplifyDataFrame=FALSE))
 }
+
+labkey.experiment.saveRuns <- function(baseUrl=NULL, folderPath, protocolName, runList)
+{
+    baseUrl=labkey.getBaseUrl(baseUrl)
+
+    ## Validate required parameters
+    if (missing(folderPath)) stop (paste("A value must be specified for folderPath."))
+    if (missing(protocolName)) stop (paste("A value must be specified for protocolName."))
+    if (missing(runList)) stop (paste("A value must be specified for runList."))
+
+    ## normalize the folder path
+    folderPath <- encodeFolderPath(folderPath)
+
+    ## Now post form with batch object filled out
```

Comment:
Very minor but this comment should probably be updated since it doesn't deal with "batch".